docs/UserGuide.md

```diff
@@ -1,186 +1,205 @@
 
-> Selects the person and loads the Google search page the person at the specified `INDEX`.<br>
-> The index refers to the index number shown in the most recent listing.<br>
-> The index **must be a positive integer** 1, 2, 3, ...
+**Dismissing a task or event, without actually completing**:
+
+Format: `dis INDEX [\\REASON]`
+
+Example:
+
+`dis 2 \\dropped the module so no longer need to complete this assignment`
+
+### 3.6. Finding event or task by keyword: `find`
+
+> Finds tasks or events whose field(s) contain any of the given keywords.<br>
+
+Format: `find KEYWORD [MORE_KEYWORDS]`
 
 Examples:
 
-* `list`<br>
-  `select 2`<br>
-  Selects the 2nd person in the address book.
-* `find Betsy` <br>
-  `select 1`<br>
-  Selects the 1st person in the results of the `find` command.
+* `find cs2103`<br>
+  Returns any task or event pertaining to cs2103
+
+### 3.7. Deleting a task/event without moving it to the expired list: `delete`
+
+> Just like edit, user can delete tasks and events in the active list by using their indexes specified in the most recent list. This option can also be used for tasks or events already in the expired list from which the user wants to get rid of.
+
+Format: `delete INDEX`
 
-### 2.8. Clearing all entries : `clear`
+### 3.8. Clearing all expired tasks and events : `clear`
 
-Clears all entries from the address book.<br>
 Format: `clear`
 
-### 2.9. Exiting the program : `exit`
+### 3.9 Viewing other material: `view`
 
-Exits the program.<br>
-Format: `exit`
+**View command history**: 
 
-### 2.10. Saving the data
+Format: `view history`
 
-Address book data are saved in the hard disk automatically after any command that changes the data.<br>
-There is no need to save manually.
+**View the calendar**: 
 
-## 3. FAQ
+Format: `view calendar`
 
-**Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with
-       the file that contains the data of your previous Address Book folder.
+**View free time during the day/week/month** : 
+
+Format: `view free day | week | month`
+
+### 3.10 Exiting the program : `exit`
+
+Format: `exit`
+
+### 3.11 Undoing the last command: `undo`
+
+Format: `undo`
 
-## 4. Command Summary
+### 3.12 Recycle 
 
-* **Add**  `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` <br>
-  e.g. `add James Ho p/22224444 e/jamesho@gmail.com a/123, Clementi Rd, 1234665 t/friend t/colleague`
+Brings back the expired tasks/events back to active list, with the deadline/event date altered.The index is as shown by `listc` command.
 
-* **Clear** : `clear`
+Format : `Recycle INDEX DEADLINE | START_DATE START_TIME to [END_DATE] END_TIME`
 
-* **Delete** : `delete INDEX` <br>
-   e.g. `delete 3`
+### 3.13 Files
 
-* **Find** : `find KEYWORD [MORE_KEYWORDS]` <br>
-  e.g. `find James Jake`
+**Switch between files**: `switchf FILEPATH`
+> if the file given by FILEPATH does not exist, `TaskCrusher` creates a new file
```

Comment:
lol! now i began to appreciate how you have come up with very non-conventional command names :-D :+1: 
i still stand by what i said earlier, try to make the commands user friendly.